rl/algorithms/Config.py

```diff
@@ -0,0 +1,23 @@
+from enum import Enum
+
+
+class Config:
+    def __init__(self) -> None:

```

Comment:
Są plany coś w inicie zawrzeć?

Jeżeli nie to jest to do kasacji

Reply:
done